docs/kick/kick.md

```diff
@@ -0,0 +1,295 @@
+# Browser beats I: synthesizing a kick drum
+
+Because I wanted to gain some experience in working with [Angular](https://angular.io/) and [NgRx](https://ngrx.io/), I started building a sample-based step sequencer that runs in the browser.
+To do that, I had to dive into the [Web Audio API](https://developer.mozilla.org/en-US/docs/Web/API/Web_Audio_API).
+I'll write something about that step sequencer later.
+First, I want to take a look at the basics of the Web Audio API and try to synthesize a kick drum.
+
+## The basis
+
+At the basis of most syntesized kick drums, there's a sine wave, or something that's close to a sine wave.
+The function below produces a sine wave with a frequency of 55Hz that plays for the duration of ten seconds.
+
+```js
+const play = () => {
+    const audioContextClass = window.AudioContext || window.webkitAudioContext;
+    const audioContext = new audioContextClass();
+
+    const sine = audioContext.createOscillator();
+    sine.type = 'sine';
+    sine.frequency.value = 55;
+
+    sine.start(audioContext.currentTime);
+    sine.stop(audioContext.currentTime + 10);
+}
+```
+
+It sounds like this:
+(You might not hear it without head phones.)
```

Comment:
This sound should be familiar to Europeans because this is the sound that AC/DC adapters make. That's because this is the same frequency your power outlet has.

Reply:
Also, not every device, especially the crappy ones, don't go down to 55Hz. 

headphones*